pkg/apis/work/v1alpha2/binding_types.go

```diff
@@ -142,6 +158,47 @@ type TargetCluster struct {
 	Replicas int32 `json:"replicas,omitempty"`
 }
 
+// GracefulEvictionTask represents an graceful eviction task.
+type GracefulEvictionTask struct {
+	// Name of the cluster where the eviction perform from.
+	Name string `json:"name"`
+
+	// Reason contains a programmatic identifier indicating the reason for the eviction.
+	// Producers may define expected values and meanings for this field,
+	// and whether the values are considered a guaranteed API.
+	// The value should be a CamelCase string.
+	// This field may not be empty.
+	// +required
+	// +kubebuilder:validation:MaxLength=32
+	// +kubebuilder:validation:MinLength=1
+	// +kubebuilder:validation:Pattern=`^[A-Za-z]([A-Za-z0-9_,:]*[A-Za-z0-9_])?$`
+	Reason string `json:"reason" protobuf:"bytes,5,opt,name=reason"`
```

Comment:
To avoid hard coding strings for `Reason` in multiple locations, you may need to enumerate valid values, or at least find a unified place to define consts.

Reply:
Done. Added a const in new push.